src/cookbook/check-status.md

```diff
@@ -11,4 +11,9 @@ head:
 
 # How to Check Iroha Status
 
-TODO
\ No newline at end of file
+```rust
+    // To get the status, the Client struct already has a function get_status(),
+    // so you may just execute it.
+    let status = iroha_client.get_status().unwrap();
+    println!("{:?}", status);
```

Comment:
We should definitely show how the result looks like

Reply:
Added the result definition in the last commit